autopush/web/webpush.py

```diff
@@ -527,6 +527,8 @@ def _router_completed(self, response, uaid_data, warning="",
                           router_type=None, vapid=None):
         """Called after router has completed successfully"""
         # Log the time taken for routing
+        if response is None:
```

Comment:
is this needed as well? I figured it was also stray debugging, seems like route_notification still always returns something

Reply:
I had originally included it because the router is technically a module that may fail. I might revisit this in the future, since you're right that we should still capture metric info off of it, but for now, it's unused code.